packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-basic-sign-out/nodejs/integrationsteps.md

```diff
@@ -6,8 +6,7 @@ The first step is to create a link, button, or other similar UI element that all
 
 <div class="common-image-format">
 
-![Sign out link](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link.png
- "Sign out link")
+![Displays a screenshot of a logout button that signs a user out of the application.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link-nodejs.png)
```

Comment:
```suggestion
![Displays a screenshot of a Logout button that signs a user out of the application.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-out-link-nodejs.png)
```

Reply:
Capitalize to be consistent with the other alt-text in the topic that refer to buttons?